src/components/sidebar/components/sidebar-nav-highlight-item/index.tsx

```diff
@@ -0,0 +1,34 @@
+import Link from 'components/link'
+import classNames from 'classnames'
+import ProductIcon from 'components/product-icon'
+import { ProductSlug } from 'types/products'
+import s from './sidebar-nav-highlight-item.module.css'
+
+/**
+ * Render a fancy-looking, product themed linked sidebar item.
+ *
+ * Intended to be used as a kind of title-ish element, to establish hierarchy
+ * within sidebar contents.
+ */
+export default function SidebarNavHighlightItem({
+	theme,
+	text,
+	href,
+	isActive,
+}: {
+	theme: ProductSlug
+	text: string
+	href: string
+	isActive?: boolean
+}) {
+	return (
+		<Link
+			aria-current={isActive ? 'page' : undefined}
+			className={classNames(s.root, s[`theme-${theme}`])}
+			href={href}
+		>
+			<ProductIcon productSlug={theme} />
+			<span className={s.text}>{text}</span>
+		</Link>
```

Comment:
💭 Should we render `SidebarNavLinkItem` here so we can more easily maintain the same functionality for things like `aria-current`, `data-heap-track`, leading icon positioning, font size/weight, a bunch of redundant & complex CSS properties?

Reply:
I don't have a strong opinion here - they felt like different enough items, with potential for further divergence, that I figured it might be slightly easier to reason about the styling if they were separate rather than intertwined, with the measured downside of some duplicate code (some differences: font weight difference, background gradient and border, brand theming prop and styling, related hover and active states, no possibility of being disabled or external or having a dropdown, no possibility of trailing icon).

That being said, happy to take another look and dive deeper into this if it feels blocking 🙇

Reply:
Nope, not blocking! Just pondering. 😊 And now I'm pondering we should get rid of that disabling... I'm almost certain that was only there for the "Coming soon" section leading up to beta! I tossed a low-priority ticket on the backlog. 😎

[[SidebarNavLinkItem] Remove ability to disable](https://app.asana.com/0/1202097197789424/1203833247481790/f)